pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -349,6 +352,11 @@ type PDSpec struct {
 	// +optional
 	StorageClassName *string `json:"storageClassName,omitempty"`
 
+	// StorageVolumes configure additional storage for PD pods.
+	// Default to the `storageClassName` storage class
```

Comment:
We can remove the description here and add desc in the struct of `StorageVolume`.